app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="top_bar">
+  <%= render partial: 'shared/menu', locals: { title: 'EXPENSES', menu_path: root_path } %>
+</div>
+
+<section class="expenses_page groups_page">
+  <div class="scroll">
+    <div class="group group_header">
+      <div>
+        <div class="header_name">
+          <% if @group.icon.attached?%>
+            <%= image_tag(@group.icon) %>
+          <%else%>
+            <%= image_tag('group.png') %>
+          <%end%>
+          <h2><%= @group.name %></h2>
+        </div>
+        <h3>$ <%= @group.expenses.sum(:amount) %></h3>
+      </div>
+      <p>Currently, the <%= @group.name %> group have following <%= @group_expenses.count %> expenses </p>
+      <button class="list_btn"><%= link_to 'Back', groups_path %></button>
+    </div>
+    
+    <div class="expeneses">
+      <% @group_expenses.each_with_index do |group_expense, index| %>
+        <div class="group">
+          <div>
+            <p><%= "#{index + 1})" %></p>
+            <% if group_expense.expense.icon.attached?%>
+              <%= image_tag(group_expense.expense.icon) %>
+            <%else%>
+              <%= image_tag('expense.png') %>
+            <%end%>
+            <div>
+              <h2><%= group_expense.expense.name %></h2>
+              <% if Time.now - group_expense.created_at <= 1.minute %>
+                <p><%= time_ago_in_words(group_expense.created_at, include_seconds: true) %> ago</p>
+              <% elsif Time.now - group_expense.created_at <= 24.hours %>
+                <p><%= time_ago_in_words(group_expense.created_at) %> ago</p>
+              <% else %>
+                <p><%= group_expense.created_at.strftime('%b %d, %Y') %></p>
+              <% end %>
+            </div>
+          </div>
+          <div class="list_gen">
+            <h3>$ <%= group_expense.expense.amount %></h3>
+            <div>
+            <button class="list_btn"><%= link_to 'Detail', group_expense_path(group_id: @group.id, id: group_expense.expense_id) %> </button>
+            </div>
+          </div>
+        </div>
+      <% end %>
+    </div>
+  </div>
+  <button class="add_btn">
+    <%= link_to 'Add new expense', new_group_expense_path(group_id: @group.id) %>
+  </button>
```

Comment:
## The same here for transaction page 💯🤩 
- [ ] Great job working on the design I think it is very great but I just have a samll note kindly align the button to be in the botton as shown in the original design since in your design it is just right after the items 🤩💯♻ so kindly align it in the bottom

- Your design
![image](https://user-images.githubusercontent.com/58553711/221380442-da74839b-bfd2-484c-b5c1-160f68b588b7.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/221380468-8b830efe-277f-4ca2-b3d4-f1c1a90d8e9d.png)